app/models/persistent_attachments/va_form.rb

```diff
@@ -0,0 +1,42 @@
+# frozen_string_literal: true
+
+class PersistentAttachments::VAForm < PersistentAttachment
+  attr_accessor :form_id
+
+  include ::FormUpload::Uploader::Attachment.new(:file)
+
+  before_destroy(:delete_file)
+
+  def max_pages
+    configs[form_id][:max_pages]
+  end
+
+  def min_pages
+    configs[form_id][:min_pages]
+  end
+
+  def warnings
+    @warnings ||= []
+  end
+
+  def as_json(options = {})
+    super(options).merge(warnings:)
+  end
+
+  private
+
+  def configs
+    {
+      '21-0779' => {
+        max_pages: 4,
+        min_pages: 2
+      }
+    }.tap do |config|
+      config.default = { max_pages: 10, min_pages: 1 }
```

Comment:
`default` could be included in the base hash. This way you don't need to call `.tap`.

Reply:
mmm yes, I just changed it. How is it now?